src/main/fc/fc_tasks.c

```diff
@@ -188,6 +189,17 @@ void taskUpdateRangefinder(timeUs_t currentTimeUs)
 }
 #endif
 
+#ifdef USE_OPTICAL_FLOW
+void taskUpdateOpticalFlow(timeUs_t currentTimeUs)
```

Comment:
Task wrapper function is not required since task is only enabled if `sensors(SENSOR_OPFLOW)` is true.